app/javascript/packages/document-capture/components/in-person-call-to-action.spec.tsx

```diff
@@ -1,18 +1,14 @@
 import sinon from 'sinon';
 import { render } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
-import { computeAccessibleDescription } from 'dom-accessibility-api';
 import { AnalyticsContextProvider } from '../context/analytics';
 import InPersonCallToAction from './in-person-call-to-action';
 
 describe('InPersonCallToAction', () => {
-  it('renders a section with label and description', () => {
+  it('renders a section with a heading', () => {
     const { getByRole } = render(<InPersonCallToAction />);
-
-    const section = getByRole('region', { name: 'in_person_proofing.headings.cta' });
-    const description = computeAccessibleDescription(section);
```

Comment:
The use of `computeAccessibleDescription` here was intended to support accessibility. Is there a good reason for removing it?

Reply:
The expected label and description (the New Feature) Tag element was removed from the component, so @tomas-nava and I opted to replace the test with one that confirmed that the section had rendered with its heading. 

The description of the removed test indicated it was checking for components being rendered within the CTA section, not that it was specifically assessing the accessibility of those rendered components. We tried to follow the pattern and confirm rendering using `computeAccessibleDescription` but were unable to get it working. We moved to instead fulfill the "components rendered" test using component content in the DOM.

If `accessibleDescription` checks are desired or required for the component, I'm happy to go back and add them at your direction.

Reply:
I tested, and this test would work if updated to use `computeAccessibleName`.

```typescript
import { computeAccessibleName } from 'dom-accessibility-api';
// ...
  it('renders a section with a heading', () => {
    const { getByRole } = render(<InPersonCallToAction />);
    const heading = getByRole('heading');
    expect(computeAccessibleName(heading)).to.equals('in_person_proofing.headings.cta');
  });
```

Reply:
Thank you!

Reply:
done in 09327aa1c7c6d8224c25d00899e5f85708a52a08